lib/Importer/ProtobufLoader.cpp

```diff
@@ -38,13 +38,9 @@ llvm::Expected<Placeholder *>
 ProtobufLoader::getOutputByName(llvm::StringRef name) const {
   RETURN_ERR_IF_NOT(
       outputVarsByName_.count(name),
```

Comment:
Can we use iterator to compare with end() to save one lookup?

Reply:
yes, it does not hurt to make it a bit more performant.